docs/reference/evercrypt/dh/index.md

```diff
@@ -0,0 +1,38 @@
+# Diffie-Hellman
+
+Diffie-Hellman key exchange on Curve25519.
+
+## API Reference
+
+```C
+#include "EverCrypt_Curve25519.h"
+```
+
+```{doxygenfunction} EverCrypt_Curve25519_scalarmult
+```
+
+Compute the scalar multiple of a point.
+
+`shared` Pointer to memory where the resulting point is written to.
```

Comment:
This is fine with me. But you sounded like you wanted to give more details. In that case I'd expect something like this
```suggestion
`shared` Pointer to 32 bytes of memory, allocated by the caller, where the resulting point is written to.
```